packages/hooks/src/hooks/useEffectAfterInitialRender.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useRef } from "react";
+
+/**
+ * Like React's useEffect, except doesn't execute on initial render.
+ */
+export const useEffectAfterInitialRender = (
```

Comment:
The need for this hook to exists feels wrong. I won't say it's wrong per se, but I've found that usually if something like this has to be done, there's a cleaner solution that can be reached some other way. For now I don't have any concrete suggestions however.

Reply:
The reason the hook is used here is to avoid unnecessary refetch when the page is loaded directly. In such case, when using regular `useEffect`, The first page is first fetched by `getServerSideProps`, and then refetched when the page mounts. IIRC I couldn't find a cleaner solution for this.